src/app/models/sellerModel.ts

```diff
@@ -0,0 +1,11 @@
+export interface SellerModel {
+    businessName: 'string';
```

Comment:
Interface are not created like this please check other example 
Egz 

export interface person {

fullname : string 
}